feature-authentication/src/main/java/com/apk/herbiary/screens/login/LoginScreen.kt

```diff
@@ -0,0 +1,104 @@
+package com.apk.herbiary.screens.login
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.rememberNavController
+import com.apk.herbiary.feature.authentication.R
+import com.apk.herbiary.screens.login.ui.AppLogo
+import com.apk.herbiary.screens.login.ui.GoogleSignInButton
+import com.apk.herbiary.screens.login.ui.PasswordTextField
+import com.apk.herbiary.screens.login.ui.SignUpButton
+
+@Composable
+fun LoginScreen(navController: NavHostController) {
+    Surface {
+        Column(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(horizontal = 24.dp)
+                .fillMaxHeight(),
+            horizontalAlignment = Alignment.CenterHorizontally,
+        ) {
+            Spacer(modifier = Modifier.height(24.dp))
+            AppLogo()
+            Spacer(modifier = Modifier.height(64.dp))
+
+            var text by remember { mutableStateOf("") }
+
+            OutlinedTextField(
+                modifier = Modifier.fillMaxWidth(),
+                value = text,
+                onValueChange = { text = it },
+                label = { Text(stringResource(id = R.string.username)) }
+            )
+
+            Spacer(modifier = Modifier.height(12.dp))
+
+
+            var password by remember { mutableStateOf("") }
+            var hasError by remember { mutableStateOf(false) }
+
+            PasswordTextField(
+                password = password,
+                onTextChange = { password = it },
+                labelText = stringResource(id = R.string.password),
+                hasError = hasError
+            )
+
+            TextButton(
+                onClick = { /*TODO*/ },
+                contentPadding = PaddingValues(2.dp),
+                modifier = Modifier
+                    .align(Alignment.End)
+                    .padding(2.dp)
+            ) {
+                Text(
+                    stringResource(R.string.forgot_password),
+                    fontSize = 12.sp,
+                    color = colorResource(id = R.color.green)
+                )
+            }
+
+            Spacer(modifier = Modifier.height(16.dp))
+
+            Button(
```

Comment:
Maybe doing only one button component with our own styles should be better so this part is shorter and can be used in all the application